docs/src/components/__tests__/back-to-top.test.js

```diff
@@ -0,0 +1,20 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import React from 'react';
+
+import BackToTop from '../footer/back-to-top';
+
+const scrollToSpy = jest.fn();
+global.scrollTo = scrollToSpy;
+
+describe('BackToTop', () => {
+	it('renders BackToTop', () => {
+		render(<BackToTop />);
+		expect(screen.getByText('Back To Top')).toBeInTheDocument();
+
+		fireEvent.scroll(window, { target: { scrollY: 100 } });
+
+		// eslint-disable-next-line testing-library/no-node-access
+		const backDom = document.getElementsByClassName('show');
```

Comment:
can you test for visibility using the testing library instead of searching for a class

https://testing-library.com/docs/guide-disappearance/#waiting-for-appearance